features/rageshake/src/main/kotlin/io/element/android/features/rageshake/logs/VectorFileLogger.kt

```diff
@@ -37,6 +39,7 @@ import java.util.logging.Logger
 class VectorFileLogger(
     context: Context,
     // private val vectorPreferences: VectorPreferences
+    private val dispatcher: CoroutineDispatcher = Dispatchers.IO,
```

Comment:
Why not injecting Dispatchers like other classes?

Reply:
The constructor is called from `TimberInitializer`, so not sure this is possible here.